threatmodel.md

```diff
@@ -10,3 +10,10 @@ It is assumed that an attacker may perform one or more the following actions:
 
 While it is not always possible to protect against all scenarios, the system should to the extent possible mitigate and/or reduce the damage caused by a successful attack, detect the occurrence of an attack and notify appropriate parties, yet remain usable for parties operating the system.  Furthermore, the system should recover from successful attacks in a way that presents low operational overhead and risk to users.
 
+Attacker Goals:
+1. Trying to have a party install a malicious image under the attackers control.
+2. Trying to have a party install an outdated image.  For example, one with known security vulnerabilities.
+3. Making images unavailable for installation.
+4. Preventing a party from learning about updates to currently installed images.
+5. Convincing a party to download large amounts of data that interfere with the party's system.
+6. Enabling future attacks of the above types to be carried out more easily.  For example, by causing a party to trust the attacker's key.
```

Comment:
```suggestion
6. Enable future attacks of the above types to be carried out more easily.  For example, by causing a party to trust the attacker's key.

## Out of Scope
The following attacks are considered out of scope for Notary v2:
1. Denial of Service (DoS) attacks.
2. Registry validation. A registry may choose to do validation when artifacts are uploaded, but this validation is out of scope of Notary v2.
```

Reply:
I wouldn't say registry validation is out of scope. There's some good ideas here. What I think we said was we'd focus on how notary would validate. In most cases, we see validation happening no the consumption of content from a registry.
But, there was some interesting ideas where a registry might validate in upload. It's not a notary goal, rather an product optiomiation potential. As long as notary can validate things, it's up to the various clients and products to implement that validation at any point they'd like.